spec/features/home_page_spec.rb

```diff
@@ -1,6 +1,45 @@
-feature 'Home Page visit', js: true do
-  scenario 'Unregistered visitor visits home page' do
-    visit '/'
-    expect(page).to have_content 'Hello World!'
+feature 'Home Page interaction', js: true do
+
+  before :each do
+    @home_page = HomePage.new
```

Comment:
Well, this is just according to site prism gem docs. And in UI test you do often need to use instance variables, especially in Cucumber features where you want to save the state between different scenarios. 

But for current needs - yes, we can I guess

Reply:
@bazarnyi yeah, I took a look at gem docs but there are always two types of people. First one uses instance variables in tests. Another one suggests using `let`s. We use `let`s because instance variables can be shared among the whole spec file and it can cause side effects.